Source/ZXing.Net.Mobile.iOS/MobileBarcodeScanner.cs

```diff
@@ -37,6 +38,13 @@ public MobileBarcodeScanner ()
 					break;
 				}
 			}
+			Initialise ();
+		}
+
+		private void Initialise() {
+			Version sv = new Version (0, 0, 0);
+			Version.TryParse (UIDevice.CurrentDevice.SystemVersion, out sv);
+			is7orgreater = sv.Major >= 7;
```

Comment:
I have realised my check is wrong here. In haste I wrote greater than or equal to 7 but in actual fact it should only be greater than 7. Do you think I could use this to check that my version is greater than 7:
```
UIDevice.CurrentDevice.CheckSystemVersion(7, 2)
```
Since there is no version 7.2 of iOS it should in theory give me the expected result.